### DISCUSSION TITLE
Vue node views

### DISCUSSION THREAD

**Discussion by silenius** [ueberdosis/tiptap#5615] [General Questions]
Hello,

Is there a way (a helper?) to render all attributes with a node view?

Thanks

**Comment by litingyes** [comment]
I think you can get the node `attrs` from `node.attrs` in props of `VueNodeViewRenderer`, then bind them yourself, such as `v-bind="node.attrs"`

**Reply by litingyes** [reply]
such as: https://github.com/vueditor/rich-text-editor/blob/cca2e60fcd88e8b5c5aaca6a3cb07f0f8bcf7069/src/editor/extensions/image/EditorImage.vue#L62

**Comment by silenius** [comment]
Thanks, I know about node.attrs but it doesn't work in this case. I have ton of attrs which render as a class, so I can't use node.attrs.class, I've to use each attr separately ..
